neutrino.go

```diff
@@ -1727,6 +1783,70 @@ func (s *ChainService) Start() error {
 		return nil
 	}
 
+	// Preload headers before starting any subsystem. Block headers should be
+	// sideloaded first as the amount of filter headers to be sideloaded
+	//// depends on it.
+	if s.blkHdrSideloader != nil {
+		log.Debugf("Sideloading block headers")
+		err := s.blkHdrSideloader.sideLoadHeaders(s.blkHdrSideloader.
+			blkHdrCurHeight,
+			s.blkHdrSideloader.blkHdrProcessor.nextCheckpt.Height, true)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	if s.cfHdrSideloader != nil {
+		_, blkTip, err := s.BlockHeaders.ChainTip()
+		if err != nil {
+			return err
+		}
+
+		s.cfHdrSideloader.fHdrLastHeight = blkTip
+		var doNotSideloadCfheaders bool
+		if s.cfHdrSideloader.fHdrSkipVerify {
```

Comment:
I think you intended this to be `! fHdrSkipVerify`?

Reply:
Thanks.